py/picca/prep_Pk1D.py

```diff
@@ -57,3 +57,16 @@ def spectral_resolution(wdisp) :
     reso = wdisp*constants.speed_light/1000.*1.0e-4*sp.log(10.)
 
     return reso
+
+def spectral_resolution_desi(reso_matrix, ll) :
+
+    dll = (ll[-1]-ll[0])/float(len(ll)-1)
+    reso= sp.clip(reso_matrix,1.0e-6,1.0e6)
+    rms_in_pixel = (sp.sqrt(1.0/2.0/sp.log(reso[len(reso)/2][:]/reso[len(reso)/2-1][:]))
+                    + sp.sqrt(4.0/2.0/sp.log(reso[len(reso)/2][:]/reso[len(reso)/2-2][:])))/2.0
```

Comment:
@Cyeche please mind the integer division in python3:

    len(reso)/2 => len(reso)//2

this will work in python 2 as well. Can you substitute `1.0/2.0 => 0.5` and `4.0/2.0 => 2`

Reply:
by the way, is it easy to understand where the formula for `rms_in_pixel` comes from? What I would have done is to reconstruct the 2D resolution matrix from the reso array (which contains a few main diagonals) and calculate the rms of lambda for each row using the resolution as a weight. Is this what this formula does?